packages/verified-fetch/src/utils/walk-path.ts

```diff
@@ -24,7 +24,9 @@ async function walkPath (blockstore: ReadableStorage, path: string, options?: Pa
   const ipfsRoots: CID[] = []
   let terminalElement: UnixFSEntry | undefined
 
-  for await (const entry of exporterWalk(path, blockstore, options)) {
+  for await (const entry of exporterWalk(path, blockstore, {
+    ...options
+  })) {
```

Comment:
undo this change if we're not changing extended's default:


```suggestion
  for await (const entry of exporterWalk(path, blockstore, options)) {
```